com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/FlowableFacade.java

```diff
@@ -254,6 +254,16 @@ public void trigger(String userId, String executionId) {
         }
     }
 
+    public void setProcessVariable(String executionId, String varName, Object varValue) {
+        processEngine.getRuntimeService()
+                .setVariable(executionId, varName, varValue);
+    }
+
+    public Object getVariable(String executionId, String varName) {
```

Comment:
Personally, I do not like this approach because the FlowableFacade is intended to hide the complex usage of processEngine with all of its services. Thus, such dummy methods here are not well welcomed in the facade.
However, there were plans to remove the facade and each call to a corresponding method from it, to be extracted to the given place. If now, someone sees these two methods, there will be assumption that each variable need to be set/retrieved using these two methods - which is not OK.

What you can do is to provide the RuntimeService where it is needed and use it there(just as we are doing with the HistoryService).